environment.yml

```diff
@@ -0,0 +1,21 @@
+###### TODO: how use exact version specifiers for all packages ######
+name: streamlit-env
+channels:
+  - conda-forge
+  - bioconda
+  - openms
+dependencies:
+  - python=3.9
+  - pyopenms
+  - openms # openms executables
+  - openms-thirdparty # e.g. third-party tools, search engines, percolator, ...
+  # TODO: if someone wants to use the nightly it might be installed in the Dockerfile with pip3 install pyopenms-nightly --extra-index-url = https://pypi.cs.uni-tuebingen.de/simple/
+  - mono # for ThermoRawFileConversion
+  - plotly
+  - pandas
+  - numpy
```

Comment:
those are transitive dependencies of pyopenms. They could be removed unless you require a specific version of them in the rest of the code.